linux/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@ set(CPPS
   lnxcon_raw.cpp
   lnxapp.cpp
   lnxcon_null.cpp
+  lnxcontroller.cpp
   lnxdata.cpp
   registry.cpp)
-#  DynXLib.cpp
 add_library(linux STATIC ${HEADERS} ${CPPS})
+add_library(lnxcontroller STATIC ${HEADERS} ${CPPS})
```

Comment:
now lnxcontroller.cpp is in the `linux` library, you don't need `lnxcontroller` library anymore

Reply:
Did I modify it correctly?

Reply:
looks fine